src/main/java/org/camunda/community/benchmarks/utils/BpmnJobTypeParser.java

```diff
@@ -0,0 +1,138 @@
+                    jobTypes.addAll(resourceJobTypes);
+                    LOG.info("Extracted {} job types from {}: {}", 
+                        resourceJobTypes.size(), resource.getFilename(), resourceJobTypes);
+                } catch (Exception e) {
+                    LOG.error("Failed to extract job types from resource {}: {}", 
+                        resource.getFilename(), e.getMessage(), e);
+                }
+            }
+        }
+
+        LOG.info("Total unique job types extracted: {}", jobTypes);
+        return jobTypes;
+    }
+
+    /**
+     * Extracts job types from a single BPMN resource.
+     */
+    private static Set<String> extractJobTypesFromResource(Resource resource, String starterId) throws IOException {
+        Set<String> jobTypes = new HashSet<>();
+        
+        try (InputStream inputStream = resource.getInputStream()) {
+            BpmnModelInstance modelInstance = Bpmn.readModelFromStream(inputStream);
+            Collection<ServiceTask> serviceTasks = modelInstance.getModelElementsByType(ServiceTask.class);
+            
+            for (ServiceTask serviceTask : serviceTasks) {
+                Collection<ZeebeTaskDefinition> taskDefinitions = serviceTask.getExtensionElements()
+                    .getElementsQuery().filterByType(ZeebeTaskDefinition.class).list();
+                
+                for (ZeebeTaskDefinition taskDef : taskDefinitions) {
+                    String type = taskDef.getType();
+                    if (type != null && !type.trim().isEmpty()) {
+                        String resolvedType = resolveJobType(type, starterId);
+                        if (resolvedType != null) {
+                            jobTypes.add(resolvedType);
+                        }
+                    }
+                }
+            }
+        }
+        
+        return jobTypes;
+    }
+
+    /**
+     * Resolves job type expressions to concrete job type strings.
+     * Only handles static types and ignores dynamic expressions.
+     */
+    private static String resolveJobType(String jobTypeExpression, String starterId) {
+        if (jobTypeExpression == null || jobTypeExpression.trim().isEmpty()) {
+            return null;
+        }
+
+        // Remove leading/trailing whitespace
+        jobTypeExpression = jobTypeExpression.trim();
+        
+        // Handle expressions like = "benchmark-task-1"
+        if (jobTypeExpression.startsWith("=")) {
+            String expression = jobTypeExpression.substring(1).trim();
+            return resolveStaticExpression(expression);
+        }
+        
+        // If it's a simple string literal (quoted), extract the content
+        if (jobTypeExpression.startsWith("\"") && jobTypeExpression.endsWith("\"") && 
+            !jobTypeExpression.contains("+")) {
+            return jobTypeExpression.substring(1, jobTypeExpression.length() - 1);
+        }
+        
```

Comment:
Let's simplify this code and simply ignore any job type that starts with `=`. We can problably remove the entire `resolveJobType` method.